Userland/Libraries/LibWeb/HTML/HTMLHyperlinkElementUtils.cpp

```diff
@@ -515,9 +515,10 @@ void HTMLHyperlinkElementUtils::follow_the_hyperlink(Optional<String> hyperlink_
         url_string = MUST(url_builder.to_string());
     }
 
-    // FIXME: 11. Let referrerPolicy be the current state of subject's referrerpolicy content attribute.
+    // 11. Let referrerPolicy be the current state of subject's referrerpolicy content attribute.
+    auto referrer_policy = ReferrerPolicy::from_string(hyperlink_element_utils_referrerpolicy()->bytes_as_string_view()).value_or(ReferrerPolicy::ReferrerPolicy::EmptyString);
```

Comment:
This causes a crash when clicking elements with no `referrerpolicy` attribute because you're getting the value of a potentially empty `Optional`. 

`ReferrerPolicy::from_string(hyperlink_element_utils_referrerpolicy().value_or({})).value_or(ReferrerPolicy::ReferrerPolicy::EmptyString);` or something equivalent should work here

Reply:
Opps - I've corrected this. Thanks for spotting :)